### DISCUSSION TITLE
How to enable embeddings mode

### DISCUSSION THREAD

**Discussion by mina58** [vllm-project/vllm#8113] [Q&A]
when i use vllm serve <model> i get a log saying that embedding_mode is false. how can i enable it to use the /v1/embedding endpoint.

**Comment by duongngyn0510** [comment]
you cannot enable `embedding_mode` in vLLM because it depends on the model architecture. you can check the models that support `embedding_mode` in vLLM by looking at `_EMBEDDING_MODELS` in the `vllm/model_executor/models/registry.py` file (tag v0.6.6) 
![image](https://github.com/user-attachments/assets/12da6693-ace8-44f4-a855-01b600228f46)
